src/main/java/net/starlark/java/eval/cpu_profiler_windows.cc

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Starlark CPU profiler stubs for unsupported platforms.
+
+#include <fcntl.h>
+#include <io.h>
+#include <jni.h>
+#include <stdlib.h>
+#include <windows.h>
+
+// We need this in order to use SetTimer and KillTimer.
+#pragma comment (lib, "User32.lib")
```

Comment:
Can you link to those libraries via the `linkopts` of `cc_binary` at [here](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/net/starlark/java/eval/BUILD;l=95?q=%22%2F%2Fconditions:default%5C%22:%20%5B%5C%22cpu_profiler_unimpl.cc%22&ss=bazel)? So that it's easier to figure out the dependencies by looking at the BUILD file.

See this [example](https://cs.opensource.google/bazel/bazel/+/master:src/main/cpp/BUILD;l=53?q=Ws2_32&ss=bazel)

Reply:
I added this by mistake